### PR TITLE
update value for one namespace.

### PR DESCRIPTION
**Pull request by morvencao**
Since we have deprecated the `istiotesting.oneNameSpace`, we need to update value for `oneNamespace` installation.

### CODE REVIEW

File: install/kubernetes/helm/istio/values-istio-one-namespace-auth.yaml [hunk 1]
```diff
@@ -15,6 +15,4 @@ global:
   # Default is 10s second
   refreshInterval: 1s
 
-istiotesting:
-  oneNameSpace: true
-
+  oneNamespace: true
```
Comment (linsun) [inline install/kubernetes/helm/istio/values-istio-one-namespace-auth.yaml]: could we leave some comment for this?  before, it is a testing param, I can safely ignore it.

Comment (morvencao) [reply install/kubernetes/helm/istio/values-istio-one-namespace-auth.yaml]: Added some comments for the `oneNamespace ` parameter.

**Pull request merged**